src/components/dev-dot-content/mdx-components/mdx-alert/types.ts

```diff
@@ -0,0 +1,16 @@
+import { InlineAlertProps } from 'components/inline-alert'
+
+export interface MdxInlineAlertProps {
+	children: InlineAlertProps['description']
+	title?: string
+	type?: 'tip' | 'highlight' | 'note' | 'warning'
```

Comment:
'success' wasn't a part of the design spec, so we can add it later if that is given to us!